bin/install.sh

```diff
@@ -27,6 +27,11 @@ COLOR_RESET="\e[0m"
 
 BASE_DIR=$(dirname "${SCRIPT_DIRECTORY}")
 
+INSTALL_INTERACTIVE=false
```

Comment:
In bash thereis no boolean. So this is implicit a sting. To clarify this behaviour it is good practice in bash to use `""` empty string for`false` and `"true"` as non-empty string for `true`.